includes/class-main.php

```
@@ -511,6 +513,75 @@ private function setup_hooks(): void {
+				$file['blockAssetsOnDemand'] = true;
+				$stored['file_optimisation'] = $file;
+				update_option( 'wppo_settings', $stored );
+
+				if ( ! isset( $this->options['file_optimisation'] ) || ! is_array( $this->options['file_optimisation'] ) ) {
+					$this->options['file_optimisation'] = array();
+				}
+				$this->options['file_optimisation']['blockAssetsOnDemand'] = true;
+			}
+
+			update_option( 'wppo_block_assets_migrated', 1 );
+		}
```

PHPMD (2.15.0)

[error] The method maybe_migrate_block_assets_setting is not named in camelCase. (undefined)

(CamelCaseMethodName)